signwriting_transcription/pose_to_signwriting/data/datasets_pose.py

```diff
@@ -12,32 +12,38 @@ def fsw_cut(fsw_text: str) -> str:
     return fsw_text
 
 
-def ms2frame(ms, frame_rate) -> int:
-    ms = int(ms)
-    return int(ms / 1000 * frame_rate)
+def ms2frame(mil_sec, frame_rate) -> int:
+    mil_sec = int(mil_sec)
+    return int(mil_sec / 1000 * frame_rate)
 
 
 def pose_to_matrix(file_path, start_ms, end_ms):
-    with open(file_path, "rb") as f:
-        pose = Pose.read(f.read())
+    with open(file_path, "rb") as file:
+        pose = Pose.read(file.read())
     frame_rate = 29.97003 if file_path == '19097be0e2094c4aa6b2fdc208c8231e.pose' else pose.body.fps
     pose = pose.body.data
     pose = pose.reshape(len(pose), -1)
     pose = pose[ms2frame(start_ms, frame_rate):ms2frame(end_ms, frame_rate)]
     return pose
 
 
-def load_dataset(folder_name):
+def load_dataset(folder_name, set_split):
     with open(f'{folder_name}/target.csv', 'r', encoding='utf-8') as csvfile:
         reader = csv.DictReader(csvfile)
 
         dataset = []
         for line in reader:
-            pose = pose_to_matrix(f"{folder_name}/{line['pose']}", line['start'], line['end'])
+            try:
+                pose = pose_to_matrix(f"{folder_name}/{line['pose']}", line['start'], line['end'])
+            except FileNotFoundError:
+                continue
             pose = pose.filled(fill_value=0)
             utt_id = line['pose'].split('.')[0]
             utt_id = f"{utt_id}({line['start']})"
-            dataset.append((utt_id, pose, fsw_cut(swu2fsw(line['text']))))
+            if not set_split:
```

Comment:
i guess what you are trying to do here is `if set_split is not None` because it is mandatory.
but can it ever be `None`? don't we always work with splits?

Reply:
The idea was to make it more generic for some cases where we don't have pre-split data, to change it so that it will return an exception if it's not the case?

Reply:
I think we should *never* have non pre-split data